### PR TITLE
Experiment with wpcom API calls using wpcom-proxy-request

### PR DESCRIPTION
**Pull request by delawski**
The basic infrastructure is added here, however the requests are silently failing. The Network tab in dev tools shows the response to the request:

```{error: "unauthorized", message: "API calls to this blog have been disabled."}```

The reason is probably the fact that the app is running on dev server and the requests are hitting the following endpoint:

```https://public-api.wordpress.com/rest/v1/sites/http%3A%2F%2Flocalhost%3A63342?http_envelope=1&check_wpcom=1```

I'll need some support preferably from someone experienced in Calypso.

### CODE REVIEW

**Comment by tyxla** [comment]
Would it make it easier for your purpose if you use this library directly, instead of trying to set up the proxy in the first place: https://github.com/Automattic/wpcom.js

**Comment by delawski** [comment]
~~I haven't tried yet. I've based this approach on one of the [Calypso modules](https://github.com/Automattic/wp-calypso/blob/e694e299b29de4bc5de12bc841940ad49ddb8941/apps/notifications/src/standalone/auth-wrapper.jsx#L60-L61).~~

**Update:**
I'm sorry for the first answer. I was too hasty and forgot that I actually tried using `wpcom` directly the last time but was getting some strange errors. I'm going to retry today.

File: src/hello-world/index.js [hunk 1]
```diff
@@ -5,18 +5,43 @@ import apiFetch from '@wordpress/api-fetch';
...
+	const [ wpcomData, setWpcomData ] = useState( [] );
 
 	useEffect( () => {
 		apiFetch( { path: '/wp/v2/posts' } ).then( ( response ) => {
 			setPosts( response );
 		} );
 	}, [] );
 
+	useEffect( () => {
+		wpcomOAuth( '67055' );
```
Comment (delawski) [inline src/hello-world/index.js]: @tyxla I'm wondering if it's safe to keep the `client_id` public? I'm not sure if it's the same property, but [in Calypso it seems to be available publicly](https://github.com/Automattic/wp-calypso/blob/10effec63507b9284037725d3f4e5ba5eac6dfbc/config/production.json#L147).

Comment (tyxla) [reply src/hello-world/index.js]: Yeah, that should be fine for now. We can just keep it in a config file, but that's just fine for a proof of concept as-is.

**Comment by delawski** [comment]
I've refined a bit the PoC. It's using `wpcom-oauth-cors` for authentication and then makes requests using `wpcom`.

It's also now taking advantage of `i18n` and `date` packages.

I've added a raw wp-admin stylesheet to quickly get the missing CSS on the page.

Screenshot 2019-10-04 at 18 29 52

**Pull request merged**